django/contrib/gis/geos/prototypes/io.py

```diff
@@ -211,7 +219,7 @@ def outdim(self, new_dim):
 
     @property
     def trim(self):
-        return self._trim
+        return default_trim_value() if self._trim is None else self._trim
```

Comment:
I have some doubts about this change. First of all, calculating `default_trim_value()` every time looks unnecessary, can we move this to a lazy module constant or something similar? Secondly, `default_trim_value()` is not taken into account in the setter, this shouldn't work :thinking: 

I'd start from removing a redundant logic in a separate PR, e.g.
```diff
diff --git a/django/contrib/gis/geos/prototypes/io.py b/django/contrib/gis/geos/prototypes/io.py
index f291ec08b6..20ef8b37ce 100644
--- a/django/contrib/gis/geos/prototypes/io.py
+++ b/django/contrib/gis/geos/prototypes/io.py
@@ -189,8 +189,7 @@ class WKTWriter(IOBase):
 
     def __init__(self, dim=2, trim=False, precision=None):
         super().__init__()
-        if bool(trim) != self._trim:
-            self.trim = trim
+        self.trim = trim
         if precision is not None:
             self.precision = precision
         self.outdim = dim
```
What do you think?

Reply:
Hi Felix,

A lazy module constant definitely looks like a good idea. 

I'll simplify the trim getter setter changes